explainer.md

```diff
@@ -508,30 +522,31 @@ The goal of user agents should be to ensure that the PEPC is not trivial to
 abuse. Therefore the user agent should consider the following potentially
 malicious tactics and mitigate them:
 
-- The site could trick the user by choosing some misleading text (e.g. "Click
-  here to proceed"). Therefore the text on the PEPC should not be able to be set
-  by the site, instead the user agent should make sure to set it to something
-  comprehensive (e.g. "Share location" for a geolocation PEPC). \
-   **Open question:**should there be a mechanism that allows the site to pick
-  one of several flavors of text (example: "Share location" vs "Use location")?
-- The style of the PEPC can be set to obscure the purpose (e.g. setting the same
-  text color and button color would make the text unreadable). Therefore the
-  style should be verified, validated and overridden by the user agent as
-  needed. More details in the [Locking the PEPC style](#locking-the-pepc-style)
-  section
-- The PEPC might be partially covered (to hide the text) with another HTML
-  element. Therefore the user agent should verify that the PEPC has been visible
-  already for some short time (e.g. 500ms or so) before it's clicked. User
-  agents that implement the
-  [IntersectionObserverV2](https://github.com/w3c/IntersectionObserver/blob/v2/explainer.md)
-  API can make use of it internally.
-- The site might try to obtain a click on the PEPC by moving it where the user
-  is about to click. Therefore the user agent should ensure that the PEPC has
-  not been moved recently (e.g. in the past 500ms or so).
-- The site might try to obtain a click on the PEPC by inserting it into the DOM
-  where the user is about to click. Therefore the user agent should ensure that
-  the PEPC has not been inserted into the DOM recently (e.g. in the past 500ms
-  or so).
+-   The site could trick the user by choosing some misleading text (e.g. "Click
+    here to proceed"). Therefore the text on the PEPC should not be able to be
+    set by the site, instead the user agent should make sure to set it to
+    something comprehensive (e.g. "Share location" for a geolocation PEPC). \
+    **Open question:**should there be a mechanism that allows the site to pick
+    one of several flavors of text (example: "Share location" vs "Use
+    location")?
+-   The style of the PEPC can be set to obscure the purpose (e.g. setting the
+    same text color and button color would make the text unreadable). Therefore
+    the style should be verified, validated and overridden by the user agent as
+    needed. More details in the
+    [Locking the PEPC style](#locking-the-pepc-style) section
+-   The PEPC might be partially covered (to hide the text) with another HTML
```

Comment:
These kinda of heuristics can quickly become an issue, as developers will constantly be trying to work around them... we end up fighting content producers at the detriment of users and their experience.